### PR TITLE
Unit test footer style(#15)

### PR DESCRIPTION
**Pull request by grleachman**

### CODE REVIEW

File: src/BlazorWebFormsComponents.Test/DataList/FlowLayout/HeaderStyleClass.razor [hunk 1]
```diff
@@ -43,6 +43,7 @@
 		//context.WaitForNextRender(() => { }, TimeSpan.FromSeconds(2));
 
 		var cut = GetComponentUnderTest();
+			Console.WriteLine(cut.Markup);
```
Comment (csharpfritz) [inline src/BlazorWebFormsComponents.Test/DataList/FlowLayout/HeaderStyleClass.razor]: I'd prefer not to have this extra Console.WriteLine in the final code.  please delete

**Comment by csharpfritz** [comment]
Looks really good.  Can we also add the FooterStyle sample to the Nav tree on the left of the Samples project?

File: src/BlazorWebFormsComponents.Test/DataList/FlowLayout/HeaderStyleClass.razor (hunk 1 above)
Comment (grleachman) [reply src/BlazorWebFormsComponents.Test/DataList/FlowLayout/HeaderStyleClass.razor]: Is there some other form of logging you'd prefer?
Without this I have to keep a copy of the website open and constantly refresh it whilst I'm doing TDD. 
The output gives me clear target to aim for, and is really useful in debugging issues, especially when I'm debugging a break on a test that I wasn't actually working on. 
By removing the output entirely I'm making it harder for the next person to figure out what's going on.
I can obviously add it back when required. I'm wondering if there's a middle ground that would be comfortable for you?

Comment (csharpfritz) [reply src/BlazorWebFormsComponents.Test/DataList/FlowLayout/HeaderStyleClass.razor]: oh wait... that's in a test, NOT the library.

We're good then.  My mistake

**Pull request merged**